src/components/usa-skipnav/usa-skipnav.stories.js

```diff
@@ -0,0 +1,23 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-skipnav.twig";
+import Data from "./usa-skipnav.json";
+
+export default {
+  title: "Components/Skipnav",
+  argTypes: {
+    modifier: {
+      control: "text",
+    },
+    label: {
```

Comment:
Consider changing the title of this control to "text" so that it is both more semantic and consistent with the rest of the controls

Reply:
I think `label` is more appropriate. We should evaluate those components with `title` further down the line.